card/mod/pointer/lib/javascript/script_pointer_config.js.coffee

```diff
@@ -24,12 +24,13 @@ decko.editorInitFunctionMap['.pointer-list-editor'] = ->
 
 $.extend decko,
   initPointerList: (input) ->
-    optionsCard = input.closest('ul').data('options-card')
-    optionsCard ||= input.closest('.pointer-list-editor').data('options-card')
-    input.autocomplete {
-      source: decko.prepUrl decko.rootPath + '/' + optionsCard +
-          '.json?view=junction_name_complete'
-    }
+    decko.initAutoCardPlete input
```

Comment:
now we put the `options-card` class directly on the input, which makes it easier to reuse this pattern.